frontend/src/components/App/Seats/index.tsx

```diff
@@ -9,26 +9,50 @@
 
 import useTranslation from 'next-translate/useTranslation';
 
+import { IConferenceStatus } from '@/jitsi/Status';
+import { useStateValue } from '@/contexts/AppContext';
 import VideoPlaceholder from '@/components/App/VideoPlaceholder';
+import SeatsStyled, { Free, Seat } from '@/components/App/Seats/styles';
 import SeatImage from '@/ui/svg/seat.svg';
+import NotAvailableImage from '@/ui/svg/unavailable-seat.svg';
 import MicMuted from '@/ui/svg/mic-muted.svg';
 import VideoMuted from '@/ui/svg/video-muted.svg';
-import SeatsStyled, { Free, Seat } from '@/components/App/Seats/styles';
 
 const Seats = () => {
   const { t } = useTranslation('app');
+  const [{ conferenceStatus }] = useStateValue();
+
+  const isIntro = conferenceStatus === IConferenceStatus.INTRODUCTION;
+  const notStarted = conferenceStatus === IConferenceStatus.NOT_STARTED;
```

Comment:
Constant (variable) names should be self explanatory

Reply:
Hey @italodr thanks, any sugestion? Maybe like `isFishbowlInIntro` `fishbowlNotStarted` ?

Reply:
That will depend on how they reference this variables in other files, but I probably will separate concepts of Fishbowl, as the Stooa event, and Conference, as the Jitsi concept of event. So, following these guidelines, I would probably use names like `CONFERENCE_INTRODUCTION` and `CONFERENCE_NOT_STARTED` (uppercase depending on internal conventions)

Reply:
Since we are creating booleans based on the comparison of the conference status with states that the app could be, they shouldn't be uppercase. And I propose as you said, use the conference keyword this way `isConferenceInIntro` and `isConferenceNotStarted` . In my pov it's pretty clear that they are `booleans` and when should they be `true` or `false`.